addons/openebs/enable

```diff
@@ -52,38 +70,36 @@ echo ""
 echo "" 
 echo "-----------------------"
 echo "" 
-echo "When using OpenEBS with a single node MicroK8s, it is recommended to use the openebs-hostpath StorageClass"
-echo "An example of creating a PersistentVolumeClaim utilizing the openebs-hostpath StorageClass"
+echo "If you are planning to use OpenEBS with multi nodes, you can use the openebs-jiva-csi-default StorageClass."
+echo "The class should be enabled as default."
+echo "An example of creating a PersistentVolumeClaim utilizing the openebs-jiva-csi-default StorageClass"
 echo "" 
 echo "" 
-echo "kind: PersistentVolumeClaim 
+echo "kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
-  name: local-hostpath-pvc
+  name: jiva-volume-claim
 spec:
-  storageClassName: openebs-hostpath
+  storageClassName: openebs-jiva-csi-default
```

Comment:
When installing, I get the storageclass name as `openebs-jiva`

```suggestion
  storageClassName: openebs-jiva
```

Reply:
I renamed to a name that makes more sense.